### ISSUE TITLE
Pneumaticcraft Sensor + RFtools Redstone compat Issue

### ISSUE THREAD

**Issue by Elvarion** [NillerMedDild/Enigmatica6#1345]
[**0.3.1.3**]

Setup
1. [Pneumaticcraft] Sensor with Entity Detector(Needs an Entity Detector Upgrade in it) that is Looking for X player (Can be Blank or looking for anything).
2. Outputting Redstone into a [RFTools] Redstone Transmitter
3. Transmitting to a [RFTools] Redstone Receiver

When a Block updates inside the range of the Sensor, The Sensor sends a Redstone Signal for some reason.

Picture 1-2
Sensor outputting into Transmitter
https://prnt.sc/1019t11
https://prnt.sc/1019u5y

Picuture 3
Receiver outputting to a [Mekanism] Logistical Sorter (Can be any block that accepts Redstone, Tried a vanilla lamp)
https://prnt.sc/1019uml

Picture 4
If you break this Birch Slab, it causes the Sensor to output redstone, which in turn turns off the lamp for a ½ second (redstone pulse)
https://prnt.sc/101arj2

**Comment by NillerMedDild** [comment]
> When a Block updates inside the range of the Sensor, The Sensor sends a Redstone Signal for some reason.

Are you picking up said broken block? :)

**Comment by Elvarion** [comment]
Whenever I broke a block, regardless of picking them up. (I was not running the pickup mode for the sensor) the sensor would pulse. But now when I went to try and check with what you said the sensor does not output a redstone signal anymore on blockbreak.

The sensor was set to
"Entity Tracker" (Third Option) >Within Range (Second) > Anything in this Filter

Im not able to replicate this anymore atm. (Wish I had made a small video showing everything setup and how it pulsed :p) And the setup has been untouched for several hours since it was used for a security system, but since it was unreliable I just left it alone.
